doc/conf.py

```diff
@@ -1,4 +1,4 @@
-from sphinx_mdolab_theme.config import *
+from sphinx_mdolab_theme.config import *  # noqa: F403
```

Comment:
I am inclined to remove the noqas because this file should now be ignored after changes to .github.